mathics/builtin/assignments/assignment.py

```diff
@@ -137,9 +137,9 @@ class Set(InfixOperator):
 
     summary_text = "assign a value"
 
-    def eval(self, lhs, rhs, evaluation):
-        "lhs_ = rhs_"
-
+    def eval(self, expr, evaluation):
+        "Pattern[expr, _ = _]"
```

Comment:
See #1440 for how to do this without introducing `Pattern[expr, ...`

Reply:
On further inspection, we don't use `expr` here at all. So the original seems better to me.

Reply:
Ok, let's return to the previous pattern.